spec/mailers/previews/terms_mailer_preview.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+# Preview with http://localhost:3000/rails/mailers/terms_mailer/
+class TermsMailerPreview < ActionMailer::Preview
+  def reminder_email
+    TermsMailer.with(user: User.first).reminder_email
+  end
+end
```

Comment:
this is a nice way to preview email formats in your browser, can do it for other mailers too as needed